src/components/data/data.js

```diff
@@ -0,0 +1,30 @@
+export let adoptionProcessItems = [
```

Comment:
this can be an object with 2 keys one for the title and the other one for the items

Reply:
done